labgrid/util/ssh.py

```diff
@@ -298,18 +299,18 @@ def _check_external_master(self):
         if check == 0:
             self._logger.debug("Found existing control socket")
             return True
-        elif b"No such file or directory" in stdout:
+        if b"No such file or directory" in stdout:
```

Comment:
This and the following change should be in one of the other commits.

Reply:
Right, I must have messed up here :/